src/main/java/models/Player.java

```diff
@@ -31,4 +57,26 @@ public boolean doubleDown(ArrayList<Card> deck){
             return true;
         }
     }
-}
+
+    public void split(){
+        playerBet = playerBet* 2;
+
+
+        split_hand.add(hand.get(hand.size()-1));
+        hand.remove(hand.get(hand.size()-1));
+        is_split = true;
+    }
+    public void split_hit(ArrayList<Card> deck){
+
```

Comment:
It seems like its a checker before it allows the hit
and I think that the hit should be before the if statement, only then the "computer" will check for if the player busted or not